misc/power.c

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file
+ * @brief Program to calculate
+ * [exponentiation](https://en.wikipedia.org/wiki/Exponentiation)
```

Comment:
```suggestion
 * [exponentiation](https://en.wikipedia.org/wiki/Exponentiation)
 *
 * @author [Du Yuanchao](https://github.com/shellhub)
```

Reply:
I think author is optional.

Reply:
It is preferably to add the author's name.